tock/employees/models.py

```diff
@@ -128,6 +131,27 @@ def organization_name(self):
 
         return ''
 
+    def is_late(self):
```

Comment:
consider this just a stub. There are no tests yet or anything. Just a way of knowing from the user object if they are late, so we can in turn flex the UI for them.

Reply:
hmm. Thinking about this functionality again, what do you think about modifying this lookup so that the default is happy tock unless the following condition is met. 

It’s the first weekday _after_ the latest reporting period less than the current date _and_ the user hasn’t submitted. 

I’m just thinking that it would be better if the default wasn’t that the user was late but rather that the user is on time. But idk.

Reply:
I like it! Most people are on time. Makes perfect sense.

Reply:
One other quick thing here - would it make sense to have this behave as a property?  It's a small semantic thing, but seeing as how this ultimately just returns `True` or `False` based on checking a couple of conditions and is not modifying anything or accepting any arguments, the `@property` decorator for the method seems appropriate to me.

Reply:
I have no real opinion either way. I don't think there's any real advantage to doing so, but I don't think there's any downside at all, either.